Data/Sys/Shaders/Anaglyph/dubois-LCD-Amber-Blue.glsl

```diff
@@ -7,11 +7,14 @@ void main()
 {
 	float4 c0 = SampleLayer(0);
 	float4 c1 = SampleLayer(1);
-	mat3 l = mat3( 1.062,-0.205, 0.299,
-	              -0.026, 0.908, 0.068,
-	              -0.038,-0.173, 0.022);
-	mat3 r = mat3(-0.016,-0.123,-0.017,
-	               0.006, 0.062, 0.017,
-	              -0.094,-0.185, 0.991);
-	SetOutput(float4(c0.rgb * l + c1.rgb * r, c0.a));
+
+	float4 lr = float4( 1.062,-0.205, 0.299,0);
+	float4 lg = float4(-0.026, 0.908, 0.068,0);
+	float4 lb = float4(-0.038,-0.173, 0.022,0);
+
+	float4 rr = float4(-0.016,-0.123,-0.017,0);
+	float4 rg = float4( 0.006, 0.062, 0.017,0);
+	float4 rb = float4(-0.094,-0.185, 0.991,0);
+
+	SetOutput(float4(dot(lr, c0) + dot(rr, c1), dot(lg, c0) + dot(rg, c1), dot(lb, c0) + dot(rb, c1), c0.a));
```

Comment:
I suppose this could be `float4(dot(lr, c0.xyz), dot(rr, c1.xyz), ...`, but most compilers should optimize this away.